src/scrabble.js

```diff
@@ -1,5 +1,130 @@
-function scrabble() {
-  // write code here
+const { LETTER_SCORES, INCORRECT_TOKENS } = require('./constants')
+let {
+  MULTIPLE,
+  VALID_SCORE,
+  MULTIPLIER_USAGE,
+  SCORE_MULTIPLIER
+} = require('./constants')
+
+const stringToArray = (word) => {
+  if (typeof word === 'string') {
+    return word.toUpperCase().split('')
+  } else {
+    return ['']
+  }
+}
+
+const stringArrayToScore = (stringArray) => {
+  let numArray = stringArray.map((val) => {
+    if (multiplyCheck(val)) {
+      MULTIPLE = scoreMultiplicatorizer(val)
+      return undefined
+    } else {
+      return findLetterScore(val)
+    }
+  })
+  numArray = numArray.filter((val) => !(val === undefined))
+  return numArray
+}
+
+const multiplyCheck = (character) => {
+  edgeCasesIDisagreeWith(character)
+  logMultiplierUsage(character)
+  if (SCORE_MULTIPLIER[character]) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const findLetterScore = (character) => {
```

Comment:
most of the functions from here to the bottom could be shortened and still readable with the shorthand for if then else: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator

Reply:
Agreed, I've seen it in use but not tried it myself yet. Something to look into for next time when dealing with just `true` / `false` conditions.